servicecertsigner/v1alpha1/types.go

```diff
@@ -14,13 +15,7 @@ import (
 type ServiceServingCertSignerConfig struct {
 	metav1.TypeMeta `json:",inline"`
 
-	// ServingInfo is the HTTP serving information for the controller's endpoints
-	ServingInfo configv1.HTTPServingInfo `json:"servingInfo,omitempty"`
-
-	// authentication allows configuration of authentication for the endpoints
-	Authentication DelegatedAuthentication `json:"authentication,omitempty"`
-	// authorization allows configuration of authentication for the endpoints
-	Authorization DelegatedAuthorization `json:"authorization,omitempty"`
+	operatorv1alpha1.GenericOperatorConfig `json:",inline"`
```

Comment:
Is this a human facing config object or the internal operator driven config object?

Reply:
Added godocs and moved to `GenericControllerConfig`